keyboards/marksard/leftover30/keymaps/default/keymap.c

```diff
@@ -0,0 +1,214 @@
+  ),
+
+  [_RAISE] = LAYOUT_all(
+  //,-----------------------------------------------------------------------------------------------------------.
+                  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_MINS,
+  //|--------+--------+--------+--------+--------+--------|--------+--------+--------+--------+--------+--------|
+               KC_LCTL, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN,   KC_UP, KC_RGHT,     _______,
+  //|--------+--------+--------+--------+--------+--------|--------+--------+--------+--------+--------+--------|
+               KC_LSFT, XXXXXXX,  KC_ESC,  KC_TAB,   KANJI,  KC_DEL, KC_COMM,  KC_DOT, KC_BSLS,     KC_ROSF,
+  //|--------+--------+--------+--------+--------+--------|--------+--------+--------+--------+--------+--------|
+      _______, _______,                                _______,                                 KC_AJST, _______
+  //`-----------------------------------------------------------------------------------------------------------'
+  ),
+
+  [_ADJUST] = LAYOUT_all(
+  //,-----------------------------------------------------------------------------------------------------------.
+                 RESET,  RGBRST, AG_NORM, AG_SWAP, XXXXXXX, KC_HOME, KC_PGDN, KC_PGUP,  KC_END,  KC_INS, KC_PSCR,
+  //|--------+--------+--------+--------+--------+--------|--------+--------+--------+--------+--------+--------|
+               RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, XXXXXXX, KC_MS_L, KC_MS_D, KC_MS_U, KC_MS_R,     KC_NLCK,
+  //|--------+--------+--------+--------+--------+--------|--------+--------+--------+--------+--------+--------|
+               RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, XXXXXXX, KC_BTN1, KC_BTN2, XXXXXXX, XXXXXXX,     KC_CAPS,
+  //|--------+--------+--------+--------+--------+--------|--------+--------+--------+--------+--------+--------|
+      _______, _______,                                _______,                                 _______, KC_CAPS
+  //`-----------------------------------------------------------------------------------------------------------'
+  )
+};
+
+uint16_t get_tapping_term(uint16_t keycode) {
+  switch (keycode) {
+    case KC_SPRA:
+      return TAPPING_LAYER_TERM;
+    default:
+      return TAPPING_TERM;
+  }
+}
+
+int RGB_current_mode;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+
+  bool result = false;
+  switch (keycode) {
+    case KANJI:
+      if (record->event.pressed) {
+        if (keymap_config.swap_lalt_lgui == false) {
+          register_code(KC_LANG2);
+        } else {
+          SEND_STRING(SS_LALT("`"));
+        }
+      } else {
+        unregister_code(KC_LANG2);
+      }
+      break;
+#ifdef RGBLIGHT_ENABLE
+    //led operations - RGB mode change now updates the RGB_current_mode to allow the right RGB mode to be set after reactive keys are released
+    case RGB_MOD:
+        if (record->event.pressed) {
+        rgblight_mode(RGB_current_mode);
```

Comment:
should use no-eeprom version here, that way, it doesn't write to it twice.
```suggestion
        rgblight_mode_noeeprom(RGB_current_mode);
```